app/views/notes/tools_places.html.erb

```diff
@@ -20,15 +20,7 @@
 
 <div class="col-md-12">
 
-  <% if params[:action] == 'methods' %>
-    <div style="text-align:center;">
-      <h1 style="font-size:3.4em;margin-top:40px;"><%= raw t('notes.tools_places.tools_and_techniques') %></h1>
-      <h4 style="color:#888;"><%= raw t('notes.tools_places.subtitle') %></h4>
-      <p><i>Public Lab's community collaboratively develops many different techniques and methods for DIY environmental monitoring. Browse many of them here and learn how to get involved.</i></p>
-    </div>
-  <% else %>
-    <h2><%= t('notes.tools_places.public_lab_chapters') %> <small><%= raw t('notes.tools_places.start_a_chapter', :url1 => "/wiki/start-a-chapter") %></small></h2>
-  <% end %>
+<%= feature('places') %>
```

Comment:
This line seems to require a little bit of indentation. It'd be great if you could fix that. And thank you for your first PR.